src/db/MigrationService.ts

```diff
@@ -48,39 +48,40 @@ export class MigrationService {
 }
 
 if (require.main === module) {
-  const config = require(resolve(process.cwd(), 'config.js')) as Config;
+  console.log('This script currently only works programmatically');
+  // const config = require(resolve(process.cwd(), 'config.js')) as Config;
```

Comment:
What's happening with these @gardnerjack ?

Reply:
It's broken. Michael has a similar issue with his so might see if he found a solution.